.github/workflows/publish-js-client.yml

```diff
@@ -3,6 +3,10 @@ name: Publish JS Client
 on:
   workflow_dispatch:
     inputs:
+      package_path:
+        description: Path to directory with package to release
+        required: true
+        type: string
```

Comment:
Could we make this a `choice` type instead so people choose the package via a dropdown instead of having to write it?

Reply:
certainly!